CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # CHANGELOG
 
+## Version 3.7.6 GA (10/28/2025)
+
+### Enhancements
+
+* Run tests against Java 25
+  ([#4495](https://github.com/microsoft/ApplicationInsights-Java/pull/4495))
+  ([#4494](https://github.com/microsoft/ApplicationInsights-Java/pull/4494))
+* Update gradle toolchain to Java 21
+  ([#4519](https://github.com/microsoft/ApplicationInsights-Java/pull/4519))
```

Comment:
I'd leave this out, users can get worried that this means we now only support Java 21+

```suggestion
```